examples/structural_mechanics/crash/conf/reader/d3plot.yaml

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 - 2025 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_target_: d3plot_reader.Reader
```

Comment:
General comment about instantiation: I recommend adding 
```yaml
_convert_: all
```
by default. This will convert OmegaConf containers into ordinary types (list, dict etc) which is more common.
[Hydra docs](https://hydra.cc/docs/advanced/instantiate_objects/overview/#parameter-conversion-strategies).